src/index.html

```diff
@@ -2,16 +2,98 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="input-body">
+        <legend>Personal infomation</legend>
+        <label class="input-block">Surname:
+          <input type="Surname">
+        </label>
+        <label class="input-block">Name:
+          <input type="Name">
```

Comment:
![image](https://user-images.githubusercontent.com/59209337/222356435-9a56e7c4-b49e-4ae4-b4b0-a2b1cd5095a9.png)

Reply:
It was not resolved. If you have any questions - feel free to ask them in `fe_chat`.